frontend/src/components/atoms/SubGoalContainer.tsx

```diff
@@ -0,0 +1,22 @@
+import { Box, Container, Heading } from '@chakra-ui/react';
+import React from 'react';
+import { SubGoal } from '../../types/ontologyTypes';
+
+type SubGoalContainerProps = {
+  subGoalNode: SubGoal;
+};
+
+const SubGoalContainer: React.FC<SubGoalContainerProps> = ({
+  subGoalNode,
+}: SubGoalContainerProps) => (
+  <Container>
+    <Box>
+      <Heading as="h3" size="md" my="4">
+        {subGoalNode.SubjectLabel}
+      </Heading>
+      <p>{subGoalNode.description}</p>
```

Comment:
```suggestion
      <Text>{subGoalNode.description}</Text>
```
When using Chakra we should avoid using normal elements as much as possible, as they do not follow the same styling.